blueocean-config/src/main/java/io/jenkins/blueocean/config/JenkinsJSExtensions.java

```diff
@@ -21,51 +21,50 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package io.jenkins.blueocean.jsextensions;
+package io.jenkins.blueocean.config;
 
 import java.io.IOException;
-import java.io.StringWriter;
+import java.io.InputStream;
 import java.net.URL;
-import java.nio.charset.Charset;
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.Enumeration;
 import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.CopyOnWriteArrayList;
 
-import org.apache.commons.io.IOUtils;
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
-import org.kohsuke.stapler.HttpResponse;
-import org.kohsuke.stapler.WebMethod;
-import org.kohsuke.stapler.verb.GET;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 import hudson.Extension;
 import hudson.PluginWrapper;
-import hudson.util.HttpResponses;
-import io.jenkins.blueocean.RootRoutable;
 import io.jenkins.blueocean.rest.model.BlueExtensionClass;
 import io.jenkins.blueocean.rest.model.BlueExtensionClassContainer;
 import jenkins.model.Jenkins;
 import net.sf.json.JSONArray;
 
 /**
  * Utility class for gathering {@code jenkins-js-extension} data.
- *
- * @author <a href="mailto:tom.fennelly@gmail.com">tom.fennelly@gmail.com</a>
  */
 @Extension
 @Restricted(NoExternalUse.class)
 @SuppressWarnings({"rawtypes","unchecked"})
-public class JenkinsJSExtensions implements RootRoutable {
+public class JenkinsJSExtensions {
```

Comment:
There are some tests that I wrote for the original version. Changes were made since and I don't think any tests were added for those changes ... I'll see about adding more.